src/ICeWall/doc/add_pad.md

```diff
@@ -0,0 +1,51 @@
+## add_pad
+
+### Synopsis
+```
+  % add_pad [-name <name>] \
```

Comment:
This does not generate an error when the same name appears twice in the list.  It should, otherwise we end up with gaps in the pad ring and null references when connecting bondpads.

Reply:
Check for repeated names added

Reply:
Could you also check if inst_name is repeated? It turns out that is the parameter we were using and it doesn't get caught by your check.

Reply:
Check added.
It occurs to me that we want the same kind of check on -signal as well to ensure that two pads dont try to use the same signal - with the exception of power/ground pads, where it is normal to have multiple pads associated with the same signal.